bin/varnishd/http1/cache_http1_proto.c

```diff
@@ -73,16 +73,23 @@ HTTP1_Headers(struct http_conn *htc)
 	AN(WS_Reservation(htc->ws));
 	assert(pdiff(htc->rxbuf_b, htc->rxbuf_e) <= WS_ReservationSize(htc->ws));
 
-	/* Skip any leading white space */
-	for (p = htc->rxbuf_b ; p < htc->rxbuf_e && vct_islws(*p); p++)
-		continue;
-	if (p == htc->rxbuf_e)
-		return (HTC_S_EMPTY);
-
-	/* Do not return a partial H2 connection preface */
-	retval = H2_prism_complete(htc);
-	if (retval != HTC_S_JUNK)
-		return (retval);
+	if (is_hdr) {
+		/* Skip any leading white space */
+		for (p = htc->rxbuf_b ; p < htc->rxbuf_e && vct_islws(*p); p++)
+			continue;
```

Comment:
Only CRLF and possibly LF need to be skipped.

Reply:
Same answer as https://github.com/varnishcache/varnish-cache/pull/4125#discussion_r2063041740.